virocon/plotting.py

```diff
@@ -477,7 +477,7 @@ def plot_2D_contour(
     x.append(x[0])
     y = coords[:, y_idx].tolist()
     y.append(y[0])
-    ax.plot(x, y, c="#BB5566")
+    ax.plot(np.asarray(x, dtype=object), np.asarray(y, dtype=object), c="#BB5566")
```

Comment:
I believe that without it, a DepreciationWarning was thrown. Here is the commit with a link to a StackOverflow post: https://github.com/virocon-organization/virocon/commit/45482e0b5ff2d21c594f0e292b3db9c971881b5c

However, by changing it back now quickly, I did not get the DepreciationWarning anymore.. strange. I was fighting with this DepreciationWarning quite a bit until I figured out how to fix it.

Reply:
Okay, the warning seems to be raised, when creating an ndarray from a list of lists, where the inner lists have different length. Why would that happen here? I think, if it does happen this solution hides the error instead of fixing the root cause.

Reply:
Hm... maybe if we have a contour that contains of multiple segments? Not sure though and I agree that the implementation might be a non-root cause fix. Should I revert the change?

Reply:
Yes, I think it would be the best to revert this change. If the warning occurs again we should investigate its cause.

Reply:
Ok, I'll revert it.